test/univariate/continuous/gamma.jl

```diff
@@ -1,3 +1,31 @@
+using Test, Distributions, OffsetArrays
+
 test_cgf(Gamma(1  ,1  ), (0.9, -1, -100f0, -1e6))
 test_cgf(Gamma(10 ,1  ), (0.9, -1, -100f0, -1e6))
 test_cgf(Gamma(0.2, 10), (0.08, -1, -100f0, -1e6))
+
+@testset "Gamma suffstats and OffsetArrays" begin
+    a = [ 2.0473208147256705, 17.82001658288441, 11.151923515093264, 5.577346651637625, 3.9024607405053313,
+        13.3630618949226, 8.626021369961277, 8.987097909704644, 9.901974166860912, 7.317596579736286,
+        1.5115770973132754]
```

Comment:
Why did you pick these values?

Reply:
I generated 11 random values from Gamma in Julia, and just pasted them here😅